styles/style.css

```diff
@@ -0,0 +1,382 @@
+* {
+    font-size: 1.15rem;
```

Comment:
a good strategy for font sizes is to use this.

```html {
  font-size: 100%;
}```

This sets the base font at 100% of the user 's settings (default is 16px) after that you should set the individual elements lke 
```
h1 {
  font-size: 2.5rem
}

h2 {
  font-size: 2rem;
}
```
etc